include/revng/Model/CommonTypeMethods.h

```diff
@@ -134,6 +178,12 @@ class CommonTypeMethods {
   model::Type &stripPointer();
   const model::Type &stripPointer() const;
 
+public:
+  /// Ensures the type this called onto is not const.
+  ///
+  /// \note This can modify some typedefs within the model: use with care.
+  void stripConstness();
```

Comment:
There is a very specific use case for it, and it's all super-duper sad. Luckily, this is temporary. Unluckily, renvg-c-side comment states that it will probably only go away post-mlir-conversion.

1. Yep, good catch, thanks!
2. This has to also go through typedefs :sadparrot:

Reply:
From the documentation, I don't understand what this method does.

Reply:
Ping

Reply:
After the related revng-c discussion this thing became even more specialized. As such, it doesn't make make sense to expose it in the first place.